### PR TITLE
Create Rule “meeting-join-info-at-the-top/rule”

### PR DESCRIPTION
**Pull request by tiagov8**
Automatically generated by Netlify CMS

### CODE REVIEW

File: rules/c/rule.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+---
+When you send a meeting appointment to someone, it’s important to put the meeting join information (e.g. Microsoft Teams Meeting URL) at the top of the email body as whoever is invited will be able to quickly find the link. 
+
+If the meeting URL is placed under the email body, there are more chances the receiver could miss out on your meeting URL.  
+
+<!--endintro-->
+
+By doing this, you will make it clear it is a remote meeting held via the tool you decided to use. This gives the receiver more time to get prepared for the meeting (e.g. Download and test a software they are not familiar with).
+
+![Figure: Bad example - The meeting link is under text body](join-info-bad.png)
```
Comment (bradystroud) [inline rules/c/rule.md:19]: Wrap in `bad` tag

```suggestion
::: bad
![Figure: Bad example - The meeting link is under text body](join-info-bad.png)
:::
```

File: rules/c/rule.md [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+
+![Figure: Good example - Always put meeting link on the top](join-info-good.png)
```
Comment (bradystroud) [inline rules/c/rule.md:21]: Wrap in `good` tag

```suggestion
::: good
![Figure: Good example - Always put meeting link on the top](join-info-good.png)
:::
```

**Review by bradystroud — CHANGES_REQUESTED** [review]

File: rules/c/rule.md [hunk 3]
```diff
@@ -0,0 +1,21 @@
+---
+type: rule
+title: Do you know to put any meeting join info at the top of the body?
+uri: meeting-join-info-at-the-top
```
Comment (bradystroud) [inline rules/c/rule.md:4]: Folder name is not the same as the URI

Comment (tiagov8) [reply rules/c/rule.md]: done - renamed folder

(I think my Ctrl C caused this 🤦‍♂️ )

**Review by bradystroud — APPROVED** [review]

**Pull request merged**